File: api/authwebservice/Models/AppUser.cs
using authwebservice.DTOs;
using System.ComponentModel.DataAnnotations.Schema;


namespace authwebservice.Models
{
    public class AppUser
    {
        public string id { get; set; }

        public string firstName { get; set; } = string.Empty;


        public string lastName { get; set; } = string.Empty;

        public string functionality { get; set; } = string.Empty;
        public bool online { get; set; } = false;

        public EntrepriseC  entreprise { get; set; }
       
       
        public string imageName { get; set; } = string.Empty;
        [NotMapped]
        public string? ImageSrc { get; set; }
    }
}

File: api/Backend_final/src/main/java/com/example/Backend_final/Entities/AppUser.java
package com.example.Backend_final.Entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;



@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class AppUser {
    private String id;
    private String firstName;
    private String lastName;
    private String functionality;
    private String ImageName;
    private Entreprise entreprise;
    private boolean online;
    private String ImageSrc;


}

